datafusion/expr/Cargo.toml

```diff
@@ -40,6 +40,7 @@ arrow = { workspace = true }
 datafusion-common = { path = "../common", version = "25.0.0" }
 lazy_static = { version = "^1.4.0" }
 sqlparser = "0.34"
+strsim = "0.10.0"
```

Comment:
I reviewed this crate and it is widely used in the ecosystem (used by clap): https://crates.io/crates/strsim but hasn't been updated for 3 years

However, it does appear to be a new dependency for datafusion,

An alternate if we are worried about this new dependency is we could inline the definition of `levenshtein` into datafusion as it is not large

https://github.com/dguo/strsim-rs/blob/65eac453cbd10ba4e13273002c843e95c81ae93f/src/lib.rs#L192-L238